Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-questions/test-questions.component.ts

```diff
@@ -110,19 +116,20 @@ export class TestQuestionsComponent implements OnInit {
      * @param category is an object of Category
      */
     selectQuestion(questionToSelect: QuestionBase, category: Category) {
-        if (questionToSelect.question.isSelect) {//If all questions are selected except one,and If user selects that question, then selectAll checkbox will be selected
-            let isAllSelected = category.questionList.every(function (question) {
-                return question.question.isSelect;
-            });
-            if (isAllSelected)
-                category.selectAll = true;
-            category.numberOfSelectedQuestion++;
-        } else {
-            category.selectAll = false;
-            questionToSelect.question.isSelect = false;
-            category.numberOfSelectedQuestion--;
+            questionToSelect.question.isSelect = true;
+            if (questionToSelect.question.isSelect) {//If all questions are selected except one,and If user selects that question, then selectAll checkbox will be selected
```

Comment:
no meaning of this if condition because you are yourself setting a true value above.